src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/booker/registry/RegisterPrisonerForBookerDto.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.booker.registry
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.NotBlank
+import java.time.LocalDate
+
+@Schema(description = "Details to register a prisoner to a booker.")
+data class RegisterPrisonerForBookerDto(
+  @JsonProperty("prisonerId")
+  @Schema(description = "Prisoner Id", example = "A1234AA", required = true)
+  @field:NotBlank
+  val prisonerId: String,
+
+  @Schema(description = "Prisoner first name", example = "James", required = true)
+  val prisonerFirstName: String,
```

Comment:
is the https://github.com/field:NotBlank annotation also needed here and for surname like prisonerId?